[2023-06-09 21:43:41,560] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: sftp_raw_data_full_load_pipeline.load_sftp_datadata_2.csv scheduled__2023-06-08T00:00:00+00:00 [queued]>
[2023-06-09 21:43:41,633] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: sftp_raw_data_full_load_pipeline.load_sftp_datadata_2.csv scheduled__2023-06-08T00:00:00+00:00 [queued]>
[2023-06-09 21:43:41,635] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2023-06-09 21:43:41,636] {taskinstance.py:1242} INFO - Starting attempt 5 of 5
[2023-06-09 21:43:41,636] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2023-06-09 21:43:41,674] {taskinstance.py:1262} INFO - Executing <Task(PythonOperator): load_sftp_datadata_2.csv> on 2023-06-08 00:00:00+00:00
[2023-06-09 21:43:41,686] {standard_task_runner.py:52} INFO - Started process 10241 to run task
[2023-06-09 21:43:41,693] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'sftp_raw_data_full_load_pipeline', 'load_sftp_datadata_2.csv', 'scheduled__2023-06-08T00:00:00+00:00', '--job-id', '54', '--raw', '--subdir', 'DAGS_FOLDER/sftp_raw_data_full_load_pipeline.py', '--cfg-path', '/tmp/tmp_1u4g8p3', '--error-file', '/tmp/tmpqn6mk7wn']
[2023-06-09 21:43:41,702] {standard_task_runner.py:77} INFO - Job 54: Subtask load_sftp_datadata_2.csv
[2023-06-09 21:43:41,793] {logging_mixin.py:109} INFO - Running <TaskInstance: sftp_raw_data_full_load_pipeline.load_sftp_datadata_2.csv scheduled__2023-06-08T00:00:00+00:00 [running]> on host 6ba608d2de6e
[2023-06-09 21:43:41,900] {taskinstance.py:1414} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=Hassan@gmail.com
AIRFLOW_CTX_DAG_OWNER=Hassan
AIRFLOW_CTX_DAG_ID=sftp_raw_data_full_load_pipeline
AIRFLOW_CTX_TASK_ID=load_sftp_datadata_2.csv
AIRFLOW_CTX_EXECUTION_DATE=2023-06-08T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-06-08T00:00:00+00:00
[2023-06-09 21:43:41,924] {logging_mixin.py:109} INFO - error while inserting to table: [Errno 2] No such file or directory: '/opt/data/temp_files/None'
[2023-06-09 21:43:41,990] {local_task_job.py:154} INFO - Task exited with return code 1
[2023-06-09 21:43:42,050] {taskinstance.py:1280} INFO - Marking task as FAILED. dag_id=sftp_raw_data_full_load_pipeline, task_id=load_sftp_datadata_2.csv, execution_date=20230608T000000, start_date=20230609T214341, end_date=20230609T214342
[2023-06-09 21:43:42,152] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
